core/jvm/src/test/scala/fs2/CompressionSuite.scala

```diff
@@ -228,8 +229,8 @@ class CompressionSuite extends Fs2Suite {
   test("deflate and inflate are reusable") {
     val bytesIn: Int = 1024 * 1024
     val chunkSize = 1024
-    val deflater = deflate[IO](bufferSize = chunkSize)
-    val inflater = inflate[IO](bufferSize = chunkSize)
+    val deflater = deflate[IO](DeflateParams(bufferSize = chunkSize))
```

Comment:
The other overload is not callable from Dotty -- appears to be a bug with overloads & default parameters, but I haven't minimized it yet.